files/build_templates/gbsyncd.service.j2

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=gbsyncd service
+Requires=database.service updategraph.service
+{% if sonic_asic_platform == 'broadcom' %}
+Requires=opennsl-modules.service
+ConditionPathExists=!/usr/share/sonic/hwsku/gearbox_config.json
+{% endif %}
+After=database.service updategraph.service
+After=interfaces-config.service
+{% if sonic_asic_platform == 'broadcom' %}
+After=opennsl-modules.service
```

Comment:
does it depends on opennsl-modules.service here?

Reply:
Done